tests/test_mf6_dis_errors.py

```diff
@@ -20,6 +22,8 @@ def test_err_unknown_var(flopy_dis_mf6):
 def test_err_cvg_failure(flopy_dis_mf6):
     """Test convergence failure (and a helper for checking I/O)"""
     mf6 = flopy_dis_mf6[1]
+    if version_tuple(mf6.get_version()) >= (6, 4, 2):
+        pytest.skip("modflow-6.4.2 or later does not raise XMIError")
     mf6.initialize()
```

Comment:
```suggestion
```

Reply:
This test should still be removed, since it didn't make too much sense to begin with.